.gitignore

```diff
@@ -0,0 +1 @@
+/target
```

Comment:
I would copy all the rust.gitignore here

Reply:
https://github.com/github/gitignore/blob/main/Rust.gitignore

Reply:
no ❤️